blocks/avm-report/avm-report.css

```diff
@@ -1,20 +1,29 @@
+main .avm-report-container.section {
+  background-color: var(--tertiary-color);
+  border-top: 1px solid var(--secondary-accent);
```

Comment:
What happens when the AVM report isn't the only thing in the section?

Reply:
Have moved this new background color option to a new class in main styles file.